test/performance/performance-tests-mako.sh

```diff
@@ -0,0 +1,310 @@
+#!/usr/bin/env bash
+
+# Copyright 2022 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This script runs the performance tests using mako sidecar stub against Knative
+# Serving built from source. It can be optionally started for each PR.
+# For convenience, it can also be executed manually.
+
+# If you already have a Kubernetes cluster setup and kubectl pointing
+# to it, call this script with the --run-tests arguments and it will use
+# the cluster and run the tests.
+
+# Calling this script without arguments will create a new cluster in
+# project $PROJECT_ID, start knative in it, run the tests and delete the
+# cluster.
+
+source $(dirname $0)/../e2e-common.sh
+
+# Skip installing istio as an add-on.
+# Temporarily increasing the cluster size for serving tests to rule out
+# resource/eviction as causes of flakiness.
+initialize --skip-istio-addon --min-nodes=10 --max-nodes=10 --perf --cluster-version=1.22 "$@"
+
+mkdir -p "${ARTIFACTS}/mako"
+echo Results downloaded to "${ARTIFACTS}/mako"
+
+# install netstat
```

Comment:
@dprotaso how about installing netstat?

Reply:
Can you make a followup issue to install netstat into our Prow image in `knative/test-infra`